selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/PullRequestPluginTest.java

```diff
@@ -115,6 +115,9 @@ public void setUp() throws Exception {
   public void tearDown() throws Exception {
     if (factoryWsName != null) {
       workspaceServiceClient.delete(factoryWsName, user.getName());
+    } else {
+      // workaround to remove test workspace for sure
```

Comment:
how many workspaces are used in this test class?

Reply:
2 workspaces: default one which is created automatically, and second one which is created during execution of the test.

Reply:
same here https://github.com/eclipse/che/pull/6354#discussion_r140168481